### PR TITLE
MCS-318 Rename color field

### PR DESCRIPTION
**Pull request by deanwetherby**
Simply renaming 'color' to 'segment_id' in ObjectMetadata. Contrasts with other branch/PR which moves segmentation color to higher level StepMetadata.

### CODE REVIEW

**Comment by rartiss55** [comment]
The run_depth_and_segmentation_test needs to get updated with the new name, otherwise LGTM.

**Comment by deanwetherby** [comment]
I'm not sure `segment_id` is the correct attribute name for instance segmentation color. It's definitely less confusing than color though.

**Comment by rartiss55** [comment]
> I'm not sure `segment_id` is the correct attribute name for instance segmentation color. It's definitely less confusing than color though.

Maybe segment_color or segmentation_color?

**Review by kdrumm — APPROVED** [review]

**Review by rartiss55 — APPROVED** [review]

File: machine_common_sense/object_metadata.py [hunk 1]
```diff
@@ -43,6 +40,9 @@ class ObjectMetadata(object):
     rotation : dict
         This object's rotation angles around the "x", "y", and "z" axes
         in degrees.
+    segment_color : dict
+        The "r", "g", and "b" pixel values of this object in images from the
+        StepMetadata's "object_mask_list".
```
Comment (bzinberg) [inline machine_common_sense/object_metadata.py:45]: This doc is super helpful and clarifying.  Thanks!

**Review by bzinberg — COMMENTED** [review]
Awesome, thanks folks

**Pull request merged**